src/pages/PingDetail.svelte

```diff
@@ -48,7 +48,7 @@
         Notification Email{ping.notification_emails.length > 1 ? 's' : ''}
       </td>
       <td>
-        {#each ping.notification_emails as email}<span>{email}</span>{/each}
+        {#each ping.notification_emails as email}<span>{email + " "}</span>{/each}
```

Comment:
A slightly more elegant way of doing this is to use `display: block`. Tailwind has a nice shorthand for this (https://tailwindcss.com/docs/display)

```suggestion
        {#each ping.notification_emails as email}<span class="block">{email}</span>{/each}
```

Reply:
Yes! Your sugession indeed makes it better. ^_^